ultraLatency.py

```diff
@@ -53,6 +53,14 @@
     print("Invalid MSPL specified: %d, must be in range [1,8]"%(options.MSPL))
     exit(1)
 
+if options.stepSize <= 0:
+    print("Invalid stepSize specified: %d, must be in range [1, %d]"%(options.stepSize, options.scanmax-options.scanmin))
+    exit(1)
+
+step = options.stepSize
```

Comment:
For lines 56-58: in case the user inputs a negative stepSize, we should print an error line, like with scanmin/max, MSPL and vt2 above.

For line 60: as mentioned earlier, to use a variable of hold the value of stepSize rather than using the direct CLI value, in order to calculate the difference between the scanmax and min if necessary.